packages/@okta/i18n/src/properties/login.properties

```
@@ -851,8 +851,8 @@ oie.verify.google_authenticator.otp.description = Enter the temporary code gener
 oie.google_authenticator.otp.enterCodeText = Enter code
 
 ## Custom OTP
-oie.verify.custom_otp.title = Verify with {0}
-oie.verify.custom_otp.description = Enter the authenticator set up code included with the device or provided by your admin.
+oie.verify.custom_otp.title = Enter a code from {0}
+oie.verify.custom_otp.description = Enter the code generated on your authenticator and verify.
```

@vincentvoong-okta - Should we add a call-to-action here instead of the vague "verify"?

```suggestion
oie.verify.custom_otp.description = Enter the code generated on your authenticator and click verify.
```

---

This string is from Vince, will merge for now, I can change later if needed